.eslintrc.json

```diff
@@ -53,7 +53,7 @@
         "func-call-spacing": [2, "never"],
         "func-names": 2,
         "func-style": [2, "declaration"],
-        "generator-star-spacing": [2, {"before": false, "after": true}],
+        "generator-star-spacing": [0, {"before": false, "after": true}],
```

Comment:
There was a bug in ESLint that causes that rule to error out on
```javascript
export async function forceLogoutIfNecessary(err, dispatch) {
```
in services/actions/helpers.js. It's supposed to be fixed in newer versions, but my local copy of ESLint keeps erroring out on it so I disabled the rule for the time being. If anyone ever tries to use generators, we can re-enable it.